Source to deb upgrade documentation incorrect

The current upgrade documentation states:

```
# upgrade to debian package-based installation
sudo make install
```

However, this does not perform an upgrade to a deb-based installation - it also doesn't actually update your progium/buildstep docker image. The correct command seems to be:

```sudo make install-from-deb```

However, it does not seem the update from deb works reliably. I'm 100% up-to-date with regards to dokku (as I just did a source update by accident - due to the documentation mistake). However, I get

```
# make install-from-deb

echo "--> Initial apt-get update"
--> Initial apt-get update
sudo apt-get update > /dev/null
sudo apt-get install -y apt-transport-https
Reading package lists... Done
Building dependency tree       
Reading state information... Done
The following packages were automatically installed and are no longer required:
  libboost-dev libboost-filesystem1.54.0 libboost-program-options1.54.0
  libboost1.54-dev libgoogle-perftools4 libsnappy1 libtcmalloc-minimal4
  libunwind8 libv8-3.14.5 mongodb-dev
Use 'apt-get autoremove' to remove them.
The following packages will be upgraded:
  apt-transport-https
1 upgraded, 0 newly installed, 0 to remove and 60 not upgraded.
Need to get 25.1 kB of archives.
After this operation, 0 B of additional disk space will be used.
Get:1 http://mirrors.linode.com/ubuntu/ trusty-updates/main apt-transport-https amd64 1.0.1ubuntu2.7 [25.1 kB]
Fetched 25.1 kB in 0s (0 B/s)                    
(Reading database ... 128025 files and directories currently installed.)
Preparing to unpack .../apt-transport-https_1.0.1ubuntu2.7_amd64.deb ...
Unpacking apt-transport-https (1.0.1ubuntu2.7) over (1.0.1ubuntu2.6) ...
Setting up apt-transport-https (1.0.1ubuntu2.7) ...
echo "--> Installing docker gpg key"
--> Installing docker gpg key
curl --silent https://get.docker.com/gpg 2> /dev/null | apt-key add - 2>&1 >/dev/null
echo "--> Installing dokku gpg key"
--> Installing dokku gpg key
curl --silent https://packagebutt.io/gpg.key 2> /dev/null | apt-key add - 2>&1 >/dev/null
echo "--> Setting up apt repositories"
--> Setting up apt repositories
echo "deb http://get.docker.io/ubuntu docker main" > /etc/apt/sources.list.d/docker.list
echo "deb https://packagebutt.io/dokku/dokku/ubuntu/ trusty main" > /etc/apt/sources.list.d/dokku.list
echo "--> Running apt-get update"
--> Running apt-get update
sudo apt-get update > /dev/null
echo "--> Installing pre-requisites"
--> Installing pre-requisites
sudo apt-get install -y linux-image-extra-`uname -r`
Reading package lists... Done
Building dependency tree       
Reading state information... Done
linux-image-extra-3.13.0-46-generic is already the newest version.
The following packages were automatically installed and are no longer required:
  libboost-dev libboost-filesystem1.54.0 libboost-program-options1.54.0
  libboost1.54-dev libgoogle-perftools4 libsnappy1 libtcmalloc-minimal4
  libunwind8 libv8-3.14.5 mongodb-dev
Use 'apt-get autoremove' to remove them.
0 upgraded, 0 newly installed, 0 to remove and 60 not upgraded.
echo "--> Installing dokku"
--> Installing dokku
sudo apt-get install -y dokku
Reading package lists... Done
Building dependency tree       
Reading state information... Done
The following packages were automatically installed and are no longer required:
  libboost-dev libboost-filesystem1.54.0 libboost-program-options1.54.0
  libboost1.54-dev libgoogle-perftools4 libsnappy1 libtcmalloc-minimal4
  libunwind8 libv8-3.14.5 mongodb-dev
Use 'apt-get autoremove' to remove them.
The following extra packages will be installed:
  buildstep rubygem-rack rubygem-rack-protection rubygem-sinatra rubygem-tilt
  sshcommand
The following NEW packages will be installed:
  buildstep dokku rubygem-rack rubygem-rack-protection rubygem-sinatra
  rubygem-tilt sshcommand
0 upgraded, 7 newly installed, 0 to remove and 60 not upgraded.
Need to get 1,315 kB of archives.
After this operation, 2,824 kB of additional disk space will be used.
Fetched 1,315 kB in 1s (1,113 kB/s)
Selecting previously unselected package rubygem-rack.
(Reading database ... 128025 files and directories currently installed.)
Preparing to unpack .../rubygem-rack_1.5.2_amd64.deb ...
Unpacking rubygem-rack (1.5.2) ...
Selecting previously unselected package rubygem-rack-protection.
Preparing to unpack .../rubygem-rack-protection_1.5.3_amd64.deb ...
Unpacking rubygem-rack-protection (1.5.3) ...
Selecting previously unselected package rubygem-tilt.
Preparing to unpack .../rubygem-tilt_1.4.1_amd64.deb ...
Unpacking rubygem-tilt (1.4.1) ...
Selecting previously unselected package rubygem-sinatra.
Preparing to unpack .../rubygem-sinatra_1.4.5_amd64.deb ...
Unpacking rubygem-sinatra (1.4.5) ...
Selecting previously unselected package buildstep.
Preparing to unpack .../buildstep_0.0.1_amd64.deb ...
Unpacking buildstep (0.0.1) ...
Selecting previously unselected package sshcommand.
Preparing to unpack .../sshcommand_0.0.1_amd64.deb ...
Unpacking sshcommand (0.0.1) ...
Setting up rubygem-rack (1.5.2) ...
Setting up rubygem-rack-protection (1.5.3) ...
Setting up rubygem-tilt (1.4.1) ...
Setting up rubygem-sinatra (1.4.5) ...
Selecting previously unselected package dokku.
(Reading database ... 128669 files and directories currently installed.)
Preparing to unpack .../dokku_0.3.17_amd64.deb ...
reload: Unknown instance: 
dpkg: error processing archive /var/cache/apt/archives/dokku_0.3.17_amd64.deb (--unpack):
 subprocess new pre-installation script returned error exit status 1
Errors were encountered while processing:
 /var/cache/apt/archives/dokku_0.3.17_amd64.deb
E: Sub-process /usr/bin/dpkg returned an error code (1)
make: *** [install-from-deb] Error 100
```

--- a/debian/preinst
+++ b/debian/preinst
@@ -29,7 +29,7 @@ case "$1" in
       } >> $NGINX_CONF
 
       rm -f /etc/nginx/sites-enabled/*
-      service nginx reload
+      service nginx reload || /etc/init.d/nginx reload
       exit 0
     fi
 

